Backpack/Calendar/Classes/BPKCalendar.h

```diff
@@ -83,6 +83,36 @@ NS_SWIFT_NAME(Calendar) @interface BPKCalendar: UIView
  */
 @property (nonatomic) NSDate *maxDate;
 
+/**
+ * The underlying scrollView's content offset
+ */
+@property (readonly) CGPoint contentOffset;
+
+/**
+ * The underlying scrollView's content inset
+ */
+@property (readonly) UIEdgeInsets contentInset;
+
+/**
+ * The underlying scrollView's content size
+ */
+@property (readonly) CGSize contentSize;
+
+/**
+ * The underlying scrollView's isDecelerating
+ */
+@property (readonly) BOOL isDecelerating;
+
+/**
+ * The underlying scrollView's isTracking
+ */
+@property (readonly) BOOL isTracking;
```

Comment:
Shouldn't this and the other `BOOL ` properties be `@property(readonly, getter=isTracking) BOOL tracking` to align with `UIScrollView`?

Reply:
What is the difference? Creating a readonly property with an overriden getter like I did will achieve the same.

Reply:
It would make it easier to make it `readwrite` later is my thinking

Reply:
No ivar is created by the compiler for that property because it is readonly and the getter is overridden.

Reply:
I am not sure if we want to make it readwrite

Reply:
These properties are readonly on UISCrollView: https://developer.apple.com/documentation/uikit/uiscrollview/1619413-istracking

Reply:
contentInset and contentSize is not intented to be modified, I think the only contentOffset could be made readwrite to scroll programmatically

Reply:
Fair enough let's merge